packages/url/src/test/index.test.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
At the moment we use a default glob base pattern to match test files which looks this way:
```js
[ '**/__tests__/**/*.js?(x)', '**/?(*.)(spec|test).js?(x)' ]
```

Jest repository uses the following highly customized mix:
```js
"testPathIgnorePatterns": [
    "/node_modules/",
    "/examples/",
    "/integration_tests/.*/__tests__",
    "\\.snap$",
    "/packages/.*/build",
    "/packages/.*/build-es5"
],
"testMatch": [
    "**/*.test.js"
]
```

In Calypso we are targeting:
```js
"testMatch": [
 	"**/bin/**/integration/*.js",
 	"**/client/**/integration/*.js",
 	"**/server/**/integration/*.js"
 ],
```

and

```js
"testMatch": [ "**/server/**/test/*.js?(x)" ],
```

I don't know what is the most popular option in WordPress world, but we should rather follow what people use :)